docs/source/techniques/DirectILL.rst

```diff
@@ -317,7 +317,7 @@ The above workflow would translate to this kind of Python script for IN4 and IN6
     DirectILLSelfShielding(
         InputWorkspace='sample',
         OutputWorkspace='corrections',
-        NumberOfSimulatedWavelengths=10
+        EventsPerPoint=10000
```

Comment:
And what about this? What's the default for `EventsPerPoint`? Why do you need to specify it now?

Reply:
The default value for EventsPerPoint is 300. For the specific calculation in the IN4 test the simulation wasn't anywhere near converged with just 300 events per point. I found that if the random seed was changed, the result changed by 10-25%. Some of the code changes in this PR are a bit like changing the random seed - the changes alter the random number sequence. 

So I've increased the number of events so that the result is less dependent on the random number sequence or the seed value.

Reply:
10-25% is very large change for this correction. In this case I would suggest to update the default value as well, since many user scripts would not have it specified.

Reply:
Also, unrelated question, was the seed fixed previously or not?

Reply:
Yes the seed value parameter has a default value of 123456789 which means the seed is fixed at this number if you don't explicitly supply a value. This sort of hides the fact the calculation isn't converged. Yes I can update the default value. The appropriate value for the number of events depends on the calculation you're running (eg what the sample shape and environment are) so 10,000 may be a bit large for some calculations. 300 is probably too small though.

Reply:
Right, 10K might be too large, note that the tests are written for IN4 and IN6, but IN5 and Panther have 250 times more pixels, which is the reason they use the sparse instrument. I think reasonable default could be 10000 for IN4 and IN6, and 1000 for IN5 and PANTHER. Do you mind implementing this using instrument parameters?

Reply:
Ahh, OK, never mind the last statement, `DirectILLSelfShielding` is an end user algorithm, so we cannot implement different defaults. In which case maybe 1000 is a good compromise?

Reply:
I can implement 1000 as the default in MonteCarloAbsorption. I think most MC absorption calculations would benefit from this increase - not just the DirectILLSelfShielding. Does that sound OK?